misc/python/materialize/scalability/endpoints.py

```diff
@@ -107,3 +108,11 @@ def up(self) -> None:
         with self.composition.override(Materialized(image=self.image)):
             self.composition.up("materialized")
             self._port = self.composition.default_port("materialized")
+
+        priv_port = self.composition.port("materialized", 6877)
+
+        priv_conn = pg8000.connect(host=self.host(), user="mz_system", port=priv_port)
+        priv_conn.autocommit = True
+        cursor = priv_conn.cursor()
+        cursor.execute("ALTER SYSTEM SET max_connections = 65535;")
+        cursor.execute("ALTER SYSTEM SET max_tables = 65535;")
```

Comment:
Optional: extract a method to specify this logic only once for `MaterializeRemote` and `MaterializeLocal`

Reply:
Thank you for keeping me honest. Refactored.